tests/tools.vitruv.applications.transitivechange.tests/src/tools/vitruv/applications/transitivechange/tests/circular/pcmumlclassjava/CompositeDataTypeConceptTest.xtend

```diff
@@ -8,11 +8,11 @@ import org.palladiosimulator.pcm.repository.Repository
 import org.palladiosimulator.pcm.repository.RepositoryFactory
 import tools.vitruv.applications.pcmjava.java2pcm.Java2PcmUserSelection
 import tools.vitruv.applications.pcmumlclass.TagLiterals
-import tools.vitruv.applications.pcmumlclass.tests.PcmUmlClassApplicationTestHelper
 
 import static org.junit.jupiter.api.Assertions.*
 import java.nio.file.Path
 import org.eclipse.uml2.uml.VisibilityKind
+import tools.vitruv.applications.pcmumlclass.tests.LegacyPcmUmlClassApplicationTestHelper
```

Comment:
Try to keep some order to the imports. Eclipse (and all other IDEs) provide functionality to do this automatically.

Reply:
Ordered in 51b18da2a02c3092ccf862c90b946e9b28e3ae6d